src/System.Security.Cryptography.OpenSsl/ref/System.Security.Cryptography.OpenSsl.csproj

```diff
@@ -1,10 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{8DEA82EF-2214-4295-8CC1-9FFB9B18838F}</ProjectGuid>
-    <Configurations>netcoreapp-Debug;netcoreapp-Release;netcoreapp3.0-Debug;netcoreapp3.0-Release</Configurations>
+    <Configurations>netcoreapp-Debug;netcoreapp-Release;netcoreapp3.0-Debug;netcoreapp3.0-Release;net47-Debug;net47-Release;netfx-Debug;netfx-Release</Configurations>
+    <AssemblyVersion Condition="'$(TargetsNetFx)' == 'true'">4.1.0.0</AssemblyVersion>
+    <!-- We need to build against net47 because that is where ECParameters got added inbox. We ship as net461 in order to not require
+    the facades when the package is restored. -->
+    <PackageTargetFramework Condition="'$(TargetGroup)' == 'net47'">net461</PackageTargetFramework>
```

Comment:
On the one hand, this feels super sketchy.  On the other hand, this entire assembly is PNSE on Windows; so it doesn't really matter :smile:.

Reply:
Yeah, I said the same thing but @ericstj suggested this because behavior-wise there wouldn't really be a change here and there is really no easy way to build this in net461.